### DISCUSSION TITLE
Docs are old?

### DISCUSSION THREAD

**Discussion by valberg** [django-tenants/django-tenants#750] [General]
I'm reading the lovely docs and saw that there is something missing in the "Useful information" section of "Using django-tenants" (https://django-tenants.readthedocs.io/en/latest/use.html#useful-information)

It looks like the latest commit to https://github.com/django-tenants/django-tenants/blob/master/docs/use.rst has not been rendered?

**Comment by tomturner** [comment]
I have just fixed that for you :)

**Reply by valberg** [reply]
Hmm, there is still nothing under "Useful information"?

**Reply by tomturner** [reply]
I get what you mean now.  Read the docs isn't auto building the docs like it should. I have pressed the magic button on read the docs so 🤞 it will work
